Playground/RxSwiftExtPlayground.playground/Pages/Index.xcplaygroundpage/Contents.swift

```diff
@@ -37,6 +37,7 @@
  - [pausableBuffered()](pausableBuffered) operator, pauses emission of elements unless the most recent element from the provided sequence is `true`; buffers elements that arrive while paused; emits buffered elements on resume.
  - [filterMap()](filterMap) operator, filters out some values and maps the rest (replaces `filter` + `map` combo)
  - [Observable.fromAsync()](fromAsync) constructor, translates an async function that returns data through a completionHandler in a function that returns data through an Observable
+ - [nwise(), pairwise()](nwise) operators, which group the items emitted by an Observable into arrays of N or pairs of consecutive items
```

Comment:
I think this logically fits better under "repeatWithBehavior" (would defer to @fpillet for confirmation)